packages/web/components/Common/Layout.js

```diff
@@ -37,10 +37,15 @@ export const Row = styled.div`
 `;
 
 export const Cell = styled.div`
-	${({ theme: { screens }, col, align }) => css`
+	${({ theme: { screens }, col, align, maxWidth }) => css`
 		flex: ${col || 1};
 		margin: 0 1rem;
 		text-align: ${align || 'left'};
+		max-width: ${maxWidth && maxWidth}rem;
```

Comment:
Dessa forma não teremos um **undefinedrem** quando a prop não for passada?

Reply:
Não, @mateus4k . Ele coloca um "max-width: none". De qualquer modo, concordo que gera dúvida. Refatorei esse pedacinho pra só usar o `max-width` caso a prop seja setada. Vlw :+1: